### PR TITLE
Don't import MARC local notes

### LINKED ISSUE #2739: Drop MARC 59X local notes fields on import
This edition: https://openlibrary.org/books/OL27794599M/Tractatus_logico-philosophicus
was imported with two local notes fields from the [MARC record](https://openlibrary.org/show-records/marc_marygrove/marygrovecollegelibrary.full.D20191108.T213022.internetarchive2nd_REPACK.mrc:202990862:1752
) which don't add any value:

> 590    $bInternet Archive - 2
> 590    $bInternet Archive 2

https://www.loc.gov/marc/bibliographic/bd59x.html

Since there's no way for us to interpret 59X fields, we should just drop them on import.

@hornc

### PR DESCRIPTION
**Pull request by hornc**
fixes #2739

Closes #

### CODE REVIEW

**Comment by hornc** [comment]
thanks @tfmorris, the change wasn't as quick as I first thought :)

Tests now updated.

**Comment by cdrini** [comment]
@hornc Could you add some info about what's changed/why? As far as I can tell this just removed a print statement, no?

File: openlibrary/catalog/marc/parse.py [hunk 1]
```diff
@@ -23,26 +23,26 @@ class SeeAlsoAsTitle(MarcException):
...
-    '700', '710', '711', '720', # contributions
-    '246', '730', '740', # other titles
-    '852', # location
-    '856'] # URL
+    '245',  # title
+    '250',  # edition
+    '260',  # publisher
+    '300',  # pagination
+    '440', '490', '830'  # series
+    ] + [str(i) for i in range(500, 588)] + [  # notes + toc + description
```
Comment (hornc) [inline openlibrary/catalog/marc/parse.py]: @cdrini this range has changed to exclude 59X fields which are local notes, and 589 which is not officially defined either.

**Comment by cdrini** [comment]
Ahhh, completely missed that change! We've talked about this in the past, but could you avoid making style changes (like double quotes to single quotes/spaces before `#`) in the middle of a PR? I honestly looked through this diff at least 3 times and didn't notice what the change was. If you want to take that on, use flake8/autopep8 to to go through all the files to fix it and then enforce it via the make file. E.g. using https://lintlyci.github.io/Flake8Rules/ , it looks like two spaces before comment is `E261`. We can enforce it by adding `E261` to the `Makefile` (quick test locally doing something like `flake8 openlibrary/plugins/upstream/addbook.py --select=E261`), and to fix them using:

```sh
pip install autopep8
autopep8 --in-place --select=E261 openlibrary/plugins/upstream/addbook.py --exclude=./.*,scripts/20*,vendor/*,*/acs4.py
```

**Pull request merged**